pom.xml

```diff
@@ -1139,10 +1139,27 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>com.linkedin.coral</groupId>
+                <artifactId>coral-hive</artifactId>
+                <version>${dep.coral.version}</version>
+            </dependency>
+
             <dependency>
                 <groupId>com.linkedin.coral</groupId>
                 <artifactId>coral-trino</artifactId>
                 <version>${dep.coral.version}</version>
+                <exclusions>
+                    <!--
+                          The trino-parser library is shaded and used internally within coral-trino library for
+                          parsing Trino statements. However, this functionality is irrelevant in the context of
+                          using coral within Trino and can be safely excluded from the dependencies of this library.
+                    -->
+                    <exclusion>
+                        <groupId>com.linkedin.coral</groupId>
+                        <artifactId>coral-trino-parser</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
We don't need to do this exclusion given `trino-parser` used in coral has been shaded.